src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.jsx

```diff
@@ -4,20 +4,33 @@ import styled from 'react-emotion';
 import {Flex, Box} from 'grid-emotion';
 import space from 'app/styles/space';
 
+const Heading = styled(Flex)`
+  margin: ${space(3)} 0;
+  flex: 1;
+`;
+
+const Title = styled(Heading)`
+  font-size: 20px;
+  font-weight: bold;
+`;
+
 class SettingsPageHeading extends React.Component {
   static propTypes = {
     icon: PropTypes.node,
     title: PropTypes.node,
     action: PropTypes.node,
     tabs: PropTypes.node,
+    noStyles: PropTypes.bool,
```

Comment:
Can we add a comment about what `noStyles` will do because it still has spacing styles.